.travis.yml

```diff
@@ -34,9 +34,12 @@ jobs:
       install:
         - npm ci
       script:
-        - npm run lint
-        - npm run check-local-changes
-        - npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
+        - npm run build
```

Comment:
In #14432 I'm proposing changes which would ensure that we always use the source code to run unit tests. If that change lands, we could do one of the following:
1. remove `npm run build` altogether from this job
2. add a flag which will allow Jest config to remove override which ensures that `build` folders aren't used

As far as I remember, if we do (2) we would have quite good coverage for all our codebase both transpiled with Babel and original source code. In development we would always use source code and on Travis we would use setup closer to what you have when using code from npm.